lookbook/docs/patterns/05-dialogs.md.erb

```diff
@@ -152,6 +148,23 @@ class TestController < ApplicationControler
 end
 ```
 
+### Dialogs that should stay open on form submission
+
+Because we normally want to close the dialog when a form is submitted successfully, we have implmented this as the default behaviour.  As described in the last section,
```

Comment:
```suggestion
Because we normally want to close the dialog when a form is submitted successfully, we have implemented this as the default behaviour.  As described in the last section,
```